modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -48,7 +48,22 @@ The endpoint is added to the message flow.
 ... From *Message type*, select the message type.
 ... From schema select *Use standard X12-<version>-<message-type>* or select *Customize* to define a custom schema.
 ... Click *Save*. +
+... From schema select *Use standard X12-<version>-<message-type>* or select *Customize* to define a custom schema.
+... Click *Save*. +
 The message type and its validation settings are added to the message flow.
+. In the *Acknowledgment* section, select the check box next to *Sent 997 functional acknowledgment* if you expect your partner to return functional acknowledgments.
+.. Click *Select*.
+.. In the *Select an endpoint: Send acknowledgment to <parter>* window, click *Select* to specify the endpoint from which to receive the acknowledgment transactions from your partner. +
```

Comment:
Please change as:

In the *Select an endpoint: Send to <parter>* window, click *Select* to specify the endpoint to which to send the acknowledgment transactions to your partner.

Reply:
Done!